### DISCUSSION TITLE
How to open klayout from the OpenLane docker shell?

### DISCUSSION THREAD

**Discussion by heaton56** [The-OpenROAD-Project/OpenLane#1465] [General]
Just getting started. 

I don't find where to setup the docker shell for klayout display and XDG_RUNTIME_DIR. 

So how do I avoid this?:

OpenLane Container (41ed003):/openlane$ klayout
Warning: QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-'
Warning: QXcbConnection: Could not connect to display 172.17.96.1:0.0
Critical: Could not connect to any X display.
OpenLane Container (41ed003):/openlane$

**Comment by vijayank88** [comment]
Check this issue: https://github.com/The-OpenROAD-Project/OpenLane/issues/1421
